Docker_files/ocsci_container/scripts/run-ocs-ci.sh

```diff
@@ -4,5 +4,15 @@ set -e
 mkdir -p /opt/cluster/logs_ocsci_container
 
 # Execute run-ci
-cd /opt/ocs-ci
-run-ci --cluster-path /opt/cluster "$@" > /opt/cluster/logs_ocsci_container/output.txt
+if [[ "$@" == *"--debug--"* ]]; then
+  cd /opt/ocs-ci-debug
+  pip3.8 install --upgrade pip setuptools
+  pip3.8 install setuptools==65.5.0
+  pip3.8 install -r requirements.txt
+  STR_TMP="$@"
+  CMD=${STR_TMP:10:5000000}
+  exec $CMD > /opt/cluster/logs_ocsci_container/output_$RANDOM.txt
+else
+  cd /opt/ocs-ci
+  exec "$@" > /opt/cluster/logs_ocsci_container/output_$RANDOM.txt
```

Comment:
Hijacking the output is not a good idea, you might want to use `tee` to have a local copy of the logs, but keep the output to stdout as is.

Reply:
done